oscarlab/graphene
# [LibOS] Add Glibc 2.34 and make it default

Add support for Glibc version 2.34.

Signed-off-by: Juro Bystricky <juro.bystricky@intel.com>



## Description of the changes 
Add support for glibc.2.34 rather than applying CVE patch(es) to Glibc-2.33



## How to test this PR? 
Run the LibOS regression test suite:
<pre>
$ cd graphene
$ openssl genrsa -3 -out ./Pal/src/host/Linux-SGX/signer/enclave-key.pem 3072
$ ISGX_DRIVER_PATH="" make SGX=1 -j$(nproc)
$ make -j$(nproc)
$ meson build --buildtype=release -Ddirect=enabled -Dsgx=enabled
$ ninja -C build
$ sudo ninja -C build install
$ cd LibOS/shim/test/regression
$ make SGX=1 regression
</pre>
Expected result: No failed tests. Something like this:
<pre>
========================================================== 103 passed, 4 skipped in 147.10 seconds ==========================================================
</pre>


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2656)
